_amenities/pools/index.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Klahanie Pools
+date: 2019-05-28 05:11
+author: Klahanie
+comments: true
+categories: [pool]
+show-title: true
+---
+<div class="alert alert-warning clearfix">Note: Klahanie Pools are closed due to the COVID-19 pandemic until further notice.
+</div>
+
+<img src="{{site.url}}images/swimming1.jpg" class="float-right col-sm-4 img-thumbnail">

```

Comment:
Do we have permission to use this photo of a minor?

Reply:
I don't know anything about the provenance of this photo. Happy to change it out if you have another one you'd prefer.